<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitedWordsTest</class>
 <widget class="QMainWindow" name="LimitedWordsTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Limited Words Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="rusWordField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>93</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctAnswerLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word in English ---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctOrWrongLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Check My Answer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation into Russian -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="engWordLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
